features/command_line/fail_fast.feature

```diff
@@ -3,17 +3,29 @@ Feature: `--fail-fast` option
   Use the `--fail-fast` option to tell RSpec to stop running the test suite on
   the first failed test.
 
-  You may also specify `--no-fail-fast` to turn it off (default behaviour).
+  You may add a parameter to tell RSpec to stop running the test suite after N
+  failed tests, for example: `--fail-fast=3`.
```

Comment:
If you make my suggested change you'll have to drop these `=` of course, but you should also mention the default is `1`.